boilerplate/frontend/supertokens-react-custom/src/pages/Auth/thirdpartypasswordless/passwordless/PasswordlessSignIn.tsx

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
Why do we need these dummy files?

Reply:
We need these dummy files because otherwise Typescript would scream that the import is invalid. As we are replacing files during runtime, these files have no particular effect at the end result.